Moco/Moco/MocoGoal/MocoContactTrackingGoal.h

```diff
@@ -60,7 +60,8 @@ class OSIMMOCO_API MocoContactTrackingGoalGroup : public Object {
 ///
 /// \f[
 /// \frac{1}{mg} \int_{t_i}^{t_f}
-///         \sum_{g \in G} \|\mathrm{proj}_{\hat{n}}(F_{m,g} - F_{e,g})\|^2 ~dt
+///         \sum_{j \in G}
+///             \|\mathrm{proj}_{\hat{n}}(\vec{F}_{m,j} - \vec{F}_{e,j})\|^2 ~dt
 /// \f]
 /// We use the following notation:
 /// - \f$ mg \f$: the total weight of the system.
```

Comment:
Is `mg` only used below? Could defer definition to be down there (this is fine either way to me, especially since this isn't technically part of the PR)

Reply:
Addressed.